python/xmlInterface.py

```diff
@@ -39,7 +40,14 @@
         Union,
     )
 except ImportError:
-    pass
+    def cast(_, o): # type: ignore
```

Comment:
For this to work in Python 2 you also need to mock all the types that you pass as the first argument of `cast`. I think a simpler option is to not use `cast` and instead just use `# type: ignore` comments where necessary with a comment stating the expected type. Then when #159 eventually happens we can use `cast` again.